app/views/errors/any.html.erb

```diff
@@ -1,11 +1,9 @@
-<% content_for :application_body do %>
```

Comment:
If you notice in accounts the error page never rendered properly (or at least hasn't rendered properly in a long time): https://accounts.openstax.org/%EB

This change makes the error actually show.